corehq/apps/userreports/models.py

```diff
@@ -707,35 +708,32 @@ class AsyncIndicator(models.Model):
     These indicators will be picked up by a queue and placed into celery to be
     saved. Once saved to the data sources, this record will be deleted
     """
-    doc_id = models.CharField(max_length=255, null=False, db_index=True)
+    doc_id = models.CharField(max_length=255, null=False, db_index=True, unique=True)
     doc_type = models.CharField(max_length=126, null=False)
     domain = models.CharField(max_length=126, null=False)
-    pillow = models.CharField(max_length=126, null=False)
     indicator_config_ids = ArrayField(
         models.CharField(max_length=126, null=True, blank=True),
         null=False
     )
     date_created = models.DateTimeField(auto_now_add=True, db_index=True)
+    date_queued = models.DateTimeField(null=True, db_index=True)
 
     class Meta(object):
-        unique_together = ('doc_id', 'pillow',)
```

Comment:
Removing a unique constraint seems a little sketchy but I think it's ok in this case as only my test domain has ever tried this feature. 

The reason I want to do this is that we can have multiple pillows add indicators for the same doc_id which should keep the queue smaller at the expense of making each task a little longer.

Also in the future I want to refactor these processors to base the cache on the doc currently being processed instead of having it in the table. This would allow N tables to have a really expensive operation (like `get_case_forms`) and only hit the db once per case change instead of N times.